tests/php/api/class-test-api-bootstrap.php

```diff
@@ -43,6 +44,18 @@ public function test_register_post_types() {
 		$this->assertTrue( post_type_exists( 'audit' ) );
 	}
 
+	/**
+	 * Test API_Bootstrap::register_roles().
+	 *
+	 * @covers ::register_roles()
+	 */
+	public function test_register_roles() {
+
+		$this->api_bootstrap->register_roles();
+		$role = get_role( 'api_client' );
+		$this->assertInstanceOf( WP_Role::class, $role );
```

Comment:
This test should be expanded on to create an `api_client` and verify `private` is working correctly.